src/main/kotlin/no/nav/syfo/personstatus/api/v2/endpoints/PersonTildelingApiV2.kt

```diff
@@ -89,5 +92,30 @@ fun Route.registerPersonTildelingApiV2(
                 call.respond(HttpStatusCode.InternalServerError)
             }
         }
+
+        get("/personer/single") {
+            try {
+                val token = getBearerHeader()
+                    ?: throw java.lang.IllegalArgumentException("No Authorization header supplied")
+                val personident = call.getPersonIdent()
+                    ?: throw IllegalArgumentException("Failed to get person resource: No $NAV_PERSONIDENT_HEADER supplied in request header")
```

Comment:
Feilmeldingen her og på linje 116 bør kanskje si noe mer spesifikt om henting av veileder-knytning for person